cmd/snap-confine/ns-support.c

```diff
@@ -131,53 +130,41 @@ void sc_reassociate_with_pid1_mount_ns(void)
 			// error.
 			return;
 		}
-		die("cannot perform readlinkat() on the mount namespace file "
-		    "descriptor of the init process");
+		die("cannot read mount namespace identifier of pid 1");
 	}
 	memset(self_buf, 0, sizeof self_buf);
 	if (readlinkat(self_mnt_fd, "", self_buf, sizeof self_buf) < 0) {
-		die("cannot perform readlinkat() on the mount namespace file "
-		    "descriptor of the current process");
+		die("cannot read mount namespace identifier of this process");
```

Comment:
Nitpick, I think s/this/current/ sounds better (but other than that +1 for the shorter message).